### PR TITLE
Add monochrome tray icon

### LINKED ISSUE #60: Support for Monochrome icon on OS X
Could you add support for monochrome icons on OS X so the icon doesn't look ugly in the menu bar. Also a HighDPI (2x) icon would be appreciated, as it looks quite pixelated on Retina displays.

### PR DESCRIPTION
**Pull request by alexalekseyenko**
fixes #60

### CODE REVIEW

File: app/main.js [hunk 1]
```diff
@@ -60,11 +60,15 @@ function displaysY (height = 600) {
 }
 
 function createTrayIcon () {
```
Comment (hovancik) [inline app/main.js]: I think only macOS should be monochrome. On Linux and Windows we should have full-color stretchly logo

Comment (alexalekseyenko) [inline app/main.js]: Haven't had a chance to test it on Linux, but that should restore the original behavior.

Comment (hovancik) [inline app/main.js]: so you think that Windows should be also monochrome?

Comment (alexalekseyenko) [inline app/main.js]: Doh! I forgot that I generated the .ico file from the monochrome icon. Thank you for catching that

Comment (hovancik) [inline app/main.js]: Thanks, tested on Linux as well and looks good!

**Pull request merged**